example/build.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/sh
```

Comment:
I don't see anything in the root `package.json` that references this. Is this example file deployed on `npm`?